samples/analyze.v1.js

```diff
@@ -180,8 +180,11 @@ async function analyzeSyntaxOfText(text) {
     type: 'PLAIN_TEXT',
   };
 
-  // Detects syntax in the document
-  const [syntax] = await client.analyzeSyntax({document});
+  // Need to specify an encodingType to receive word offsets
+  const encodingType = 'UTF-8';
```

Comment:
is this what we document in the API, interestingly in Node.js' own libraries you usually write `utf8`.

Reply:
Good catch!

The API actually specifies "UTF8", no hyphen. 
https://cloud.google.com/natural-language/docs/reference/rest/v1/EncodingType